scio-core/src/main/scala/com/spotify/scio/values/SCollection.scala

```diff
@@ -55,6 +56,8 @@ import scala.util.Try
 /** Convenience functions for creating SCollections. */
 object SCollection {
 
+  private[values] val logger = LoggerFactory.getLogger(this.getClass)
```

Comment:
```suggestion
  private[values] val Logger = LoggerFactory.getLogger(this.getClass)
```

`PairSCollectionFunctions` is also lower case can you change it?